scripts/2-defineCredentials.sh

```diff
@@ -24,7 +24,7 @@ clear
 echo "==================================================================="
 echo -e "${YLW}Please enter the values as requested below: ${NC}"
 echo "==================================================================="
-read -p "Dynatrace Tenant ID (8-digits) (current: $DT_TENANT_ID) : " DT_TENANT_ID_NEW
+read -p "Dynatrace Tenant ID (foo.dynatracedomain.com) (current: $DT_TENANT_ID) : " DT_TENANT_ID_NEW
```

Comment:
adjust the "read" statement to say ""Dynatrace Tenant Base URL".  Remove example in parenthesis so that the output formatting has all the values line up.

Reply:
I can change the variable names, but we can't include https:// in the parameter because the tenant ID is used to populate the manifests/istio/service_entity_tmpl ENDPOINT 
`  - match:
    - port: 443
      sni_hosts:
      - ENDPOINT_PLACEHOLDER
    route:
    - destination:
        host: ENDPOINT_PLACEHOLDER
        port:
          number: 443
          name: HTTPS`

The code in the creatIstionServiceEntry.sh file is the following, where $1 is the expected tenant ID without 'https://'

`cat ../manifests/istio/service_entries_tpl/part1 >> se_tmp/service_entries_oneagent.yml

echo -e "  - $1" >> se_tmp/hosts
cat ../manifests/istio/service_entry_tmpl | sed 's~ENDPOINT_PLACEHOLDER~'"$1"'~' >> se_tmp/service_entries

for row in $entries; do
    row=$(echo $row | sed 's~https://~~')
    row=$(echo $row | sed 's~/communication~~')
    echo -e "  - $row" >> se_tmp/hosts
    cat ../manifests/istio/service_entry_tmpl | sed 's~ENDPOINT_PLACEHOLDER~'"$row"'~' >> se_tmp/service_entries
done`

Reply:
There are also references to the base url without 'https://' in both 
/manifests/jenkinsk8s-jenkins-deployment.yml
/manifests/jenkins/ocp-jenkins-deployment.yml

Reply:
Istio is not used.  I need to remove that code & subfolder.  If you notice here, it is commented out.
https://github.com/dt-kube-demo/setup-infra/blob/master/scripts/4-setupInfrastructure.sh#L100

I think it is more intuitive to ask a student to cut-n-paste the URL for Dynatrace.  I don't think it would be every http vs http.

If we ever add back in istio, then we can remove the http prefix just for that script.

Reply:
There's still the Jenkins issue (comment just before your reply above)
I think if we leave it without 'https://' and include the example in the define credentials script (aligned, now, of course), we'll be good. Otherwise, you'll have to manipulate the data in another script and track which scripts need to manipulate and which don't.

Reply:
yes in the DefineCreds script we would want to have it say if it can fit...(Most users of the lab will be using trial tentants)

```read -p "Dynatrace Host (e.g gan16820.live.dynatrace.com) ..."```

Reply:
ok, I think it's all set. I updated the example and parameter names. 
I looks like they are visible in this pull, so I don't think I have to do anything else.